spec/src/main/asciidoc/architecture.adoc

```diff
@@ -61,14 +61,20 @@ Required base metrics are exposed under `/metrics/base`.
 
 ===== Application metrics
 
-Application specific metrics can not be baked into the server as they are supposed to be provided by the
+Application specific metrics can not be baked into the server as they are _supposed_ to be provided by the
```

Comment:
It seems incorrect to say "application specific metrics can not be baked into the server", but then have things like FT and REST metrics essentially doing that.  Perhaps we could say "Application specific metrics are provided either by the application directly, through use of the Java API, or provided on behalf of the application, for example when applications use REST or MicroProfile Fault Tolerance APIs."